sdk/servicebus/azure-servicebus/tests/test_message.py

```diff
@@ -237,8 +237,7 @@ def test_amqp_message():
     amqp_annotated_message = AmqpAnnotatedMessage(sequence_body=[1, 2, 3])
     body = [sequence for sequence in amqp_annotated_message.body]
     assert amqp_annotated_message.body_type == AmqpMessageBodyType.SEQUENCE
-    assert len(body) == 1
-    assert body[0] == [1, 2, 3]
```

Comment:
are there any pyAMQP unit tests we can add as well ?

Reply:
we have send/receive tests I'm not sure how else this could be captured from the amqp layer as its in the sdk

Reply:
let me check the existing send